yarn-project/prover-node/src/factory.ts

```diff
@@ -34,6 +36,7 @@ export async function createProverNode(
   options: {
     prefilledPublicData?: PublicDataTreeLeaf[];
   } = {},
+  services: NamespacedApiHandlers = {} as NamespacedApiHandlers,
```

Comment:
Not a fan of this change. The `NamespacedApiHandlers` are currently all handled in the `aztec/src/cli/cmds`, and I'd rather keep them all in one place.

Reply:
Right, yeah I see what you mean

Reply:
Done